View/ListItems/SongInfo.cs

```diff
@@ -13,6 +13,7 @@ namespace View
     public class SongInfo
     {
         public Song Song { get; set; }
+        public Artist Artist { get; set; }

```

Comment:
Staat Artist niet al in Song?

Reply:
klopt
maar die kun je niet ophalen in de songlist.xaml op lijn 17
daar voor zou je weer het lazy loading nodig hebben

Reply:
Heb ik dat inmiddels niet al anders gefixt?
Ik dacht dat ik die gewoon altijd ophaal tijdens het ophalen van een nummer?

Reply:
nope

Reply:
staat niks in song dbmodel

Reply:
Okaaaay, dat ga ik wel een keer fixen.